aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClientStore.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2017-2021 Amazon.com, Inc. or its affiliates.
+ * All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.amazonaws.mobile.client;
+
+import com.amazonaws.internal.keyvaluestore.AWSKeyValueStore;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.locks.ReadWriteLock;
```

Comment:
Are these imports still in the other file? Can you run optimize imports on the old file?

Reply:
yep